engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -1037,4 +1037,59 @@ namespace dmGameSystem
         pit->m_FnIterateNext = CompMeshIterPropertiesGetNext;
     }
 
+    static dmGameObject::Result CompMeshTypeCreate(const dmGameObject::ComponentTypeCreateCtx* ctx, dmGameObject::ComponentType* type)
+    {
+        MeshContext* mesh_context = new MeshContext;
```

Comment:
The mesh context can now be moved to within this .cpp file.

Reply:
Ok. Nice...